src/main/java/com/labsynch/labseer/db/migration/postgres/V2_0_1_4__set_defaults.java

```diff
@@ -243,7 +241,8 @@ private void loadCorpNames(JdbcTemplate jdbcTemplate) throws FileNotFoundExcepti
 		// or generate it from a simple counter
 		int numCorpnames = jdbcTemplate.queryForInt("SELECT COUNT(*) FROM pre_def_corp_name");
 		if (numCorpnames < 1L){
-			boolean usePredefinedList = mainConfig.getServerSettings().isUsePredefinedList();
+			// boolean usePredefinedList = mainConfig.getServerSettings().isUsePredefinedList();
```

Comment:
Good find. I temporarily commented this out to get the code compile at the beginning of this work.  I missed that it was still commented out. I'll try and fix rather than remove.

Reply:
Oh, ha. I didn't realize that the `usePredefinedList` was hardcoded to false long before this change. I'll remove this code